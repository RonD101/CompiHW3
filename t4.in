bool a() {
    return true;
}


void main() {

    if(




        x < 2
    )
}