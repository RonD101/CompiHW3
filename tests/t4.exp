---end scope---
line 3: identifier func is already defined
