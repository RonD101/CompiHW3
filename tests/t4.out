1
---end scope---
3
line 3: identifier func is already defined
